tests/src/test/java/alluxio/server/ft/journal/JournalShutdownIntegrationTest.java

```diff
@@ -178,15 +178,24 @@ public void singleMasterMountUnmountJournal() throws Exception {
 
   @Test
   public void multiMasterMountUnmountJournal() throws Exception {
-    MultiMasterLocalAlluxioCluster cluster = setupMultiMasterCluster();
-    UnderFileSystemFactory factory = mountUnmount(cluster.getClient());
-    // Kill the leader one by one.
-    for (int kills = 0; kills < TEST_NUM_MASTERS; kills++) {
-      cluster.waitForNewMaster(120 * Constants.SECOND_MS);
-      assertTrue(cluster.stopLeader());
+    MultiMasterLocalAlluxioCluster cluster = null;
+    UnderFileSystemFactory factory = null;
+    try {
+      cluster = new MultiMasterLocalAlluxioCluster(TEST_NUM_MASTERS);
+      cluster.initConfiguration();
+      cluster.stop();
```

Comment:
Why can we stop the cluster here?

Reply:
Nice catch. It should have been stopLeader.